config/config.schema.json

```diff
@@ -620,7 +620,27 @@
           },
           "subscription": {
             "type": "string"
-          }
+          },
+          "istio": {
```

Comment:
With the three different version attributes, does it makes sense to add some description field?

Reply:
Honestly, unless proven otherwise, description field should be expected for all IMO

Reply:
description added.